src/content/staking/withdrawals/index.md

```diff
@@ -0,0 +1,165 @@
+
+### Validator "sweeping" {#validator-sweeping}
+
+When a validator is scheduled to propose the next block, it is required to build a withdrawal queue, of up to 16 eligible withdrawals. This is done by originally starting with validator index 0, determining if there is an eligible withdrawal for this account per the rules of the protocol, and adding it to the queue if there is. The validator set to propose the following block will pick up where the last one left off, progressing in order indefinitely.
+
+<InfoBanner emoji="🕛">
+Think about an analogue clock. The hand on the clock points to the hour, progresses in one direction, doesn’t skip any hours, and eventually wraps around to the beginning again after the last number is reached.<br/><br/>
+Now instead of 1 through 12, imagine the clock has 0 through N <em>(the total number of validator accounts that have ever been registered on the Beacon Chain, over 500,000 as of Jan 2023).</em><br/><br/>
+The hand on the clock points to the next validator that needs to be checked for eligible withdrawals. It starts at 0, and progresses all the way around without skipping any accounts. When the last validator is reached, the cycle continues back at the beginning.
+</InfoBanner>
+
+#### Checking an account for withdrawals {#checking-an-account-for-withdrawals}
+
+While a proposer is sweeping through validators for possible withdrawals, each validator being checked is evaluated against a short series of questions to determine if a withdrawal should be triggered, and if so, how much ETH should be withdrawn.
+
+1. **Has a withdrawal address been provided?** If no withdrawal address has been provided, the account is skipped and no withdrawal initiated.
+2. **Is the validator exited and withdrawable?** If the validator has fully exited, and we have reached the epoch where their account is considered to be "withdrawable", then a full withdrawal will be processed. This will transfer the entire remaining balance to the withdrawal address.
+3. **Is the effective balance maxed out at 32?** If the account has withdrawal credentials, is not fully exited, and has rewards above 32 waiting, a partial withdrawal will be processed which transfers only the rewards above 32 to the user's withdrawal address.
+
+There are only two actions that are taken by validator operators during the course of a validator's life cycle that influence this flow directly:
+
+- Provide withdrawal credentials to enable any form of withdrawal
+- Exit from the network, which will trigger a full withdrawal
+
+### Gas free {#gas-free}
+
+This approach to staking withdrawals avoids requiring stakers to manually submit a transaction requesting a particular amount of ETH to be withdrawn. This also means there is **no gas (transaction fee) required**, and withdrawals also do not compete for existing execution layer block space.
+
+### How soon will I get my rewards payments? {#how-soon}
```

Comment:
```suggestion
### How frequently will I get my staking rewards? {#how-soon}
```

this section is in my opinion more important than how withdrawals work and i would argue should be moved up under When staking withdrawals are enabled